README.md

```diff
@@ -144,6 +148,51 @@ go run ./cmd/lvh run --image _data/images/base.qcow2
 go run ./cmd/lvh run --image _data/images/base.qcow2 --qemu-disable-kvm
 ```
 
+### Cilium development workflow
+
+Set up an LVH VM based on the kind image:
+
+```bash
+./scripts/pull_image.sh quay.io/lvh-images/kind:bpf-next-main
+lvh run --image $(ls -Art images/*.qcow2 | tail -n 1) --host-mount /path/to/cilium
+```
+
+Then, log into the VM as root and setup some dependencies:
+
+```bash
+# DNS
+echo "nameserver 1.1.1.1" > /etc/resolv.conf
```

Comment:
These steps line up with the steps I had written down based on my local testing.